lib/solr/commit/request.rb

```diff
@@ -0,0 +1,15 @@
+module Solr
+  module Commit
+    class Request
+      include Solr::Support::ConnectionHelper
+      PATH = '/update'.freeze
+
+      def run
+        # the way to do commit message in SOLR is to send an empty 
+        # request with ?commit=true in the URL.
+        raw_response = connection(PATH, commit: true).get
```

Comment:
Is it possible to use `post` instead of `get` for this?

Reply:
My concern is that usually `get` is not used for requests that have side effects. Also this could cause some problems because of http request caching.